File: php-slim/lib/Model/RequestTimeMapArrivalSearch.php
<?php
/**
 * RequestTimeMapArrivalSearch
 */
namespace OpenAPIServer\Model;

/**
 * RequestTimeMapArrivalSearch
 */
class RequestTimeMapArrivalSearch
{
    
    /** @var string $id */
    private $id;
    
    /** @var \OpenAPIServer\Model\Coords $coords */
    private $coords;
    
    /** @var \OpenAPIServer\Model\RequestTransportation $transportation */
    private $transportation;
    
    /** @var int $travelTime */
    private $travelTime;
    
    /** @var \DateTime $arrivalTime */
    private $arrivalTime;
    
    /** @var \OpenAPIServer\Model\RequestTimeMapProperty[] $properties */
    private $properties;
    
    /** @var \OpenAPIServer\Model\RequestRangeNoMaxResults $range */
    private $range;
}

File: fsharp-giraffe-server/traveltimeplatform/src/model/RequestTimeMapArrivalSearch.fs
namespace traveltimeplatform.Model

open System
open System.Collections.Generic
open traveltimeplatform.Model.Coords
open traveltimeplatform.Model.RequestRangeNoMaxResults
open traveltimeplatform.Model.RequestTimeMapProperty
open traveltimeplatform.Model.RequestTransportation

module RequestTimeMapArrivalSearch = 

  //#region RequestTimeMapArrivalSearch


  type RequestTimeMapArrivalSearch = {
    Id : string;
    Coords : Coords;
    Transportation : RequestTransportation;
    TravelTime : int;
    ArrivalTime : DateTime;
    Properties : RequestTimeMapProperty[];
    Range : RequestRangeNoMaxResults;
  }
  //#endregion
  